Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mozillamarketplace/centos-python27-mkt:0.5
```

Comment:
As we chatted its cached. We've already got busybox which is tiny, may as well use that.